perfmetrics/scripts/read_cache/mount_gcsfuse.sh

```diff
@@ -34,7 +34,7 @@ max_size_in_mb=100
 cache_location=/tmp/read_cache/
 bucket_name=gcsfuse-read-cache-fio-load-test
 stat_or_type_cache_ttl_secs=6048000 # 168h or 1 week
-stat_cache_capacity=4096
+stat_cache_max_size_mb=2
```

Comment:
You need to change, line: 48 and line 26 accordingly.

Reply:
@raj-prince Do you mean convert from stat-cache capacity passed down with `-m` to stat-cache `size-mb` value ? Or to change the description of `-m` to mean stat-cache size-mb ?

Reply:
Done both.